locales/en/manage-kafka-permissions.json

```diff
@@ -16,22 +16,17 @@
     "all": "All",
     "alter": "Alter",
     "alter_configs": "Alter configs",
+    "aria_label": "Resource type operation value",
```

Comment:
What is this string for?
`aria-label` is a common html attribute that can be appear for many different elements. Using this as an id for a string isn't specific enough. 

Including "value" isn't needed here. It's like including the word "button" as part of the button text.

Reply:
Oh, ignore my first question. I expanded the context of this rule, and see it's nested under `"operations"`. 🤦‍♀️

Reply:
For my second comment, further down, I see examples where the aria-label is `"... {{value}}"`. Is that what's intended here?